### PR TITLE
Add LeanAndGreenRecipes

### LINKED ISSUE #443: Please add Leanandgreenrecipes.net
Please add leanandgreenrecipes.net as a scraper option.

https://leanandgreenrecipes.net/recipes/mexican/yellow-squash-taco-casserole/

### PR DESCRIPTION
**Pull request by wengtad**
closes #443

### CODE REVIEW

File: recipe_scrapers/leanandgreenrecipes.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        cuisine = self.schema.cuisine()
+        return cuisine[cuisine.find(">") + 1 :].replace("</a>", "")
+
+    def description(self):
+        return (
+            self.schema.description()
+            .replace("</p> <p>", "\n")
+            .replace("<p>", "")
+            .replace("</p>", "")
+        )
```
Comment (jayaddison) [inline recipe_scrapers/leanandgreenrecipes.py:51]: Out of curiosity: are you able to find any counter-example recipes that _do not_ require this kind of escaping of the schema description (and/or `recipeCuisine` field)?

Comment (wengtad) [reply recipe_scrapers/leanandgreenrecipes.py:53]: thanks for reviewing, provided a fix, please check.

: wonder how  to add the `hacktoberfest-accepted` label into this merge request?

Comment (hhursev) [reply recipe_scrapers/leanandgreenrecipes.py:53]: I'll remember you @wengtad and will attach `hacktoberfest-accepted` labels to your PRs through October 😉

Comment (wengtad) [reply recipe_scrapers/leanandgreenrecipes.py:53]: Thanks!

**Pull request merged**